src/main/java/de/muenchen/isi/domain/service/BauvorhabenService.java

```diff
@@ -140,7 +148,7 @@ public AbfrageModel assignBauvorhabenToAbfrage(@Nullable final UUID bauvorhabenI
      * Diese Methode soll dann verwendet werden, um die beim Mapping verloren gegangene Information zum Bauvorhaben wieder in der Infrastruktureinrichung einzusetzen.
      * Der Parameter 'bauvorhabenId' darf null sein. In diesem Fall passiert nichts.
      *
-     * @param bauvorhabenId id des {@link BauvorhabenModel}s. Darf null sein.
+     * @param bauvorhabenId            id des {@link BauvorhabenModel}s. Darf null sein.
```

Comment:
Doppelter Text. Bereits in der Beschreibung oben drüber dass es null sein kann.
```suggestion
     * @param bauvorhabenId id des {@link BauvorhabenModel}s
```